### DISCUSSION TITLE
Do not restart docker container instance when failing

### DISCUSSION THREAD

**Discussion by mkazinauskas** [coollabsio/coolify#4035] [Feature Requests]
Hello,
Currently, if the docker container instance is failing it is being automatically restarted. 
I need functionality to run once and without any restarts afterward. 
I want to trigger the container manually and run it once.

This functionality is required for database migration scripts. I want to trigger a Docker container with a DB migration script once, and when it finishes, it should be shut down and not restarted.

**Comment by mkpanq** [comment]
+1
Found related question, maybe will help: https://github.com/coollabsio/coolify/discussions/3447
But I'm not fan of this solution - when I deploy my migration container using docker-compose, I need to make changes using "Connect to predefined networks" and even then, I can't find proper hostname for my postgres service :(

**Comment by GaNiziolek** [comment]
Related question: https://github.com/coollabsio/coolify/discussions/8669
